styles/pageStyles/trips.module.css

```diff
@@ -57,7 +57,34 @@
     padding-top: 2rem;
 }
 
+.tripHeader {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
 
+.createTripBtn{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    position: absolute;
+    width: 100%;
+    max-width: 300px;
+    height: 68px;
+    left: calc(50% - 300px/2);
+    top: 70%;
+    background: #FC266B;
+    border-radius: 10px;
+    color: white;
+    font-family: 'Geliat';
```

Comment:
def kill the font family -- we use another font already throughout the whole project